packages/api/lib/rulesHelpers.js

```diff
@@ -104,6 +104,44 @@ function lookupCollectionInEvent(eventObject) {
   });
 }
 
+/*
```

Comment:
```suggestion
/**
```

This will allow the TypeScript compiler (and VS Code when viewing files) to use the JSDocs to infer types.